src/ExadelMentorship.WebApi/Program.cs

```diff
@@ -16,10 +12,28 @@
 
 builder.Services.AddBlServices();
 
-
 var app = builder.Build();
 
 // Configure the HTTP request pipeline.
+
+app.UseExceptionHandler(new ExceptionHandlerOptions
+{
+    ExceptionHandler = (c) =>
+    {
+        var exception = c.Features.Get<IExceptionHandlerFeature>();
+        var statusCode = exception.Error.GetType().Name switch
```

Comment:
* Mind to address the warnings? They are ruining the image of true beauty.
* You don't wanna make `Program.cs` bloated with this code. Move it somewhere else, keep `Program.cs` tidy and succinct.

Reply:
done